### DISCUSSION TITLE
Can't join server

### DISCUSSION THREAD

**Discussion by Sneezy123** [LanternPowered/Terre#10] [General]
Can you please make a tutorial for "dummies" on which things you should change like the IPs in the config.yaml and how to join the server in both mobile and on pc version (crossplay). It would best if it was a video tutorial. 
Thanks in regards!
![Screenshot_20211108-175909_Terraria~2](https://user-images.githubusercontent.com/83452014/140789174-770bec18-8936-43f9-b358-021e927838fe.jpg)
![Screenshot_20211108-175924_Terraria~3](https://user-images.githubusercontent.com/83452014/140789177-d556bb2b-a908-413b-96c4-9b4e46f5ddbc.jpg)
![Screenshot_20211108-175932_Terraria~3](https://user-images.githubusercontent.com/83452014/140789180-11151b40-16c3-4def-b591-a49d7ea4c902.jpg)

**Comment by Cybermaxke** [comment]
Looks like your on the right track, currently the Terraria server used needs to match de mobile version, because there are features in 1.4.2.3 that aren't supported yet on the mobile version, so you need to run a 1․4․0․5 desktop server instead. I may improve that in the future.

As for a video tutorial, I will first finish up the wiki and then see if it's still needed, did you already checkout my video on my portfolio: https://www.seppevolkaerts.be/ (scroll down and click on Terre). You can see a setup with two servers there, and I also show the config briefly (config uses a different format now, but the principle is the same).

Also, no need to create a new discussion for every related question, just continue in this discussion.  ;)

**Reply by Sneezy123** [reply]
I watched that already. But thanks a lot
